Dockerfile

```diff
@@ -324,4 +345,4 @@ COPY rootfs/make_iso.sh /
 
 RUN /make_iso.sh
 
-CMD ["cat", "boot2docker.iso"]
+CMD ["sh", "-c", "[ -t 1 ] && exec bash || exec cat boot2docker.iso"]
```

Comment:
so much this!! 😄 

![gif-keyboard-18384011143327871983](https://cloud.githubusercontent.com/assets/541832/14693616/c8822928-0760-11e6-9e2b-66049c24c698.gif)